setup/docker.md

```diff
@@ -158,6 +158,40 @@ multi-machine scaling.
 
 For organizations, we recommend one Docker host per team of 5-10 developers.
 
+## Docker Compose
```

Comment:
this was something I had suggested before (https://github.com/coder/docs/pull/818#discussion_r795902639), I suppose if we're good with the additional maintenance then more options is always good

cc @Emyrk